tests/components/shadow.test.js

```diff
@@ -17,11 +17,11 @@ suite('shadow component', function () {
       });
       el.setAttribute('shadow', {});
       mesh = new THREE.Mesh(
-        new THREE.Sphere(2),
+        new THREE.SphereGeometry(2),
         new THREE.MeshBasicMaterial({color: 0xffff00})
       );
       meshWithMaterialArray = new THREE.Mesh(
-        new THREE.Sphere(2),
+        new THREE.SphereGeometry(2),
```

Comment:
`THREE.Sphere` exists but the arguments are center vec3 and radius number in latest threejs... I spend really too much time to find that the cryptic error was because of this lol. `THREE.SphereGeometry` first arg is radius number.